packages/react/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -6,11 +7,12 @@ import { Button } from '../Button/Button';
 export type DropdownItemProps = Omit<ButtonProps, 'variant' | 'size' | 'color'>;
 
 export const DropdownItem = forwardRef<HTMLButtonElement, DropdownItemProps>(
-  function DropdownItem({ className, style, ...rest }, ref) {
+  function DropdownItem({ className, ...rest }, ref) {
+    /* All attributes are set on the <Button>, since li is possible to style from <Dropdown.List> className  */
     return (
-      <li className={className} style={style}>
+      <li>
         <Button
-          className='ds-dropdown__item'
+          className={cl('ds-dropdown__item', className)}
```

Comment:
This goes agains our rule of adding `style` and `className` to the outermost element

Reply:
I disagree a bit - I understand the point if there is one element (such as `<Button>`) but in this case it is a sub-component, and you can easily reach `<li>` by adding `className` to `Dropdown.List` - then it even natural to change to `.my-list > li `.

But if the className is on li and not button , e.g. `.my-dropdown-item { line-height: 3rem }` will have no effect, since in reality it is li that you are styling, even if you meant button (which might be an a as well, so then the consumer has to do: `.my-dropdown-item > :is(a, button, [role="button"])` - which feels less natural - especially when all other props end up on another element.